### ISSUE TITLE
Delphi XE5 needed to be added!

### ISSUE THREAD

**Issue by odyright** [crypto2011/IDR#46]

**Comment by crypto2011** [comment]
Try knowledge base for XE4

**Comment by odyright** [comment]
I tried but it doesn't recreate all the unit.pas of the project i want to decompile. And some procedures and functions aren't correct/readable.

**Comment by odyright** [comment]
Please i need help, how to use KBBUILDER for generating the **Delphi XE5** KB..?

**Comment by piratesephiroth** [comment]
I don't think kbbuilder supports delphi xe5 yet.

**Comment by odyright** [comment]
so! could someone send me the "**kb2015.bin**" & "**syskb2015.bin**" files..?

**Comment by crypto2011** [comment]
[odyright](/odyright)
Just copy dcu you want to add to knowledge base, copy dcu32int to this  directory and run as 
dcu32int -K * -U, rename kb.bin to kb2015.bin

**Comment by odyright** [comment]
thank you @crypto2011 but what did you mean by "**Just copy dcu you want to add to knowledge base?**", where to find them? in that path: "**Program Files (x86)\Embarcadero\Studio\20.0\lib\win32\release ?**"

**Comment by crypto2011** [comment]
Yes, if you see dcu-files there. syskb2015.bin must contain only 2 files (system.dcu and sysinit.dcu). kb2015.bin usually contains most popular units, used by programmers. You can decide what units you will add.

**Comment by greenozon** [comment]
@odyright
did you have some success?
if yes - please share it to the public!

**Comment by odyright** [comment]
sry @greenozon for the delay. No i figured with of errors in my file.

**Comment by Onelio** [comment]
Hi, I would like to add that for me it worked removing the space between K and * like dcu32int -K* -U
I figured it out after debugging with Delphi 7 to find the errors.
